pyanaconda/core/constants.py

```diff
@@ -486,3 +486,7 @@ class PayloadRequirementType(Enum):
 
 # Path to the System Purpose configuration file on a system.
 RHSM_SYSPURPOSE_FILE_PATH = "/etc/rhsm/syspurpose/syspurpose.json"
+
+# GID and UID status
+ID_USE_VALUE = "USE VALUE"
+ID_USE_DEFAULT = "USE DEFAULT"
```

Comment:
I would use the names `ID_MODE_USE_VALUE` and `ID_MODE_USE_DEFAULT` to avoid possible confusion. Please, use underscores instead of spaces in the values. It is pretty much our convention.